### PR TITLE
MEN-4435: Build and publish mender-configure Debian package

### PR DESCRIPTION
**Pull request by lluiscampos**
And add test to verify mender-connect and mender-configure deb packages.

### CODE REVIEW

File: docker-mender-dist-packages [hunk 1]
```diff
@@ -48,10 +48,21 @@ for arch in amd64 armhf arm64; do
             --volume $(pwd)/output:/output \
             $IMAGE_NAME:cross-${arch} \
             mender-connect \
-            github.com/mendersoftware/mender-connect \
+            https://github.com/mendersoftware/mender-connect \
             ${MENDER_CONNECT_VERSION:-master} \
             ${arch}
 
+    # Build mender-configure deb package (only for native arch)
+    if [ "${arch}" = "amd64" ]; then
```
Comment (tranchitella) [inline docker-mender-dist-packages:56]: Looks good, but I don't get why we build the mender-configure deb package only on amd64.

**Review by tranchitella — APPROVED** [review]

File: docker-mender-dist-packages (hunk 1 above)
Comment (lluiscampos) [reply docker-mender-dist-packages:56]: We use `Architecture: all` in control file, meaning that we need to build it only once. This `if` clause here is a bit miss-leading: what it is trying to do is making sure we build it only once on the native architecture (`amd64`). The generated `.deb` package has a suffix `...._all.deb`

Does it make sense?

Comment (tranchitella) [reply docker-mender-dist-packages:56]: yes it does, the comment is a bit misleading, though

Comment (lluiscampos) [reply docker-mender-dist-packages:56]: Rephrased :+1:

**Review by kacf — APPROVED** [review]
Fabio is better than me at reviewing the deb internal stuff, but looks good on the surface.

**Pull request merged**